react/src/pages/ResourcesPage.tsx

```diff
@@ -71,10 +70,22 @@ const ResourcesPage: React.FC<ResourcesPageProps> = (props) => {
         </Suspense>
       ) : null}
       {curTabKey === 'resourceGroup' ? (
-        // @ts-ignore
-        <backend-ai-resource-group-list active />
+        <>
+          <Suspense
+            fallback={
+              <Skeleton
+                active
+                style={{ padding: token.paddingContentVerticalLG }}
+              />
+            }
+          >
+            <ResourceGroupList />
+          </Suspense>
+          {/* @ts-ignore */}
+          {/* <backend-ai-resource-group-list active /> */}
+        </>
```

Comment:
```suggestion
          <Suspense
            fallback={
              <Skeleton
                active
                style={{ padding: token.paddingContentVerticalLG }}
              />
            }
          >
            <ResourceGroupList />
          </Suspense>
```

Reply:
It resolved at the top of the PR stack #3636